ext/mbstring/tests/mb_http_input.phpt

```diff
@@ -5,7 +5,7 @@ mbstring
 --POST--
 a=���ܸ�0123456789���ܸ쥫�����ʤҤ餬��
 --GET--
-b=���ܸ�0123456789���ܸ쥫�����ʤҤ餬��
+b=%C6%FC%CB%DC%B8%EC0123456789%C6%FC%CB%DC%B8%EC%A5%AB%A5%BF%A5%AB%A5%CA%A4%D2%A4%E9%A4%AC%A4%CA
```

Comment:
Shouldn't the UTF-8 passing be fixed instead?

Reply:
Ok, ~UTF-8~ EUC-JP, but where is the real problem, shouldn't the binary/EUC-JP passing be fixed instead?

Reply:
In [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986), reserved character is available. That means we can not use the binary in URI. Therefore, I think fix these test files.

Reply:
But given the tests pass on linux, is this limitation really in Windows and not in php-src?

Reply:
However,  perhaps I miss any, I'll investigate next morning. Thank you.